src/components/config/dc.js

```diff
@@ -3,11 +3,13 @@ export const DC_PIE_OUTER_RADIUS = 100;
 export const DC_PIE_WIDTH = 220;
 export const DC_PIE_HEIGHT = 220;
 
-export const DC_COUNTLABEL_SIZE = 36;
+export const DC_COUNTLABEL_SIZE = 38;
 export const DC_COLORS = {
   GREY: '#E2DFDF',
   PURPLE: '#7B1A6A',
   GREEN: '#38954C',
   ORANGE: '#d36141',
   WHITE: '#FFF',
+  BLUE_BRAND: '#3489ca',
+  LIGHT_PURPLE: '#954788',
```

Comment:
I think we should reuse the colour palette defined in ot-ui https://github.com/opentargets/ot-ui/blob/master/lib/constants.js

Reply:
Thanks @mirandaio, I missed that ot-ui constants. Yes we can refactor those values into the colour palette.